justfile

```diff
@@ -68,36 +68,52 @@ install-targets:
 # Generates the docs
 generate-docs:
     cargo clean --doc
-    cargo doc --features "{{all_features}}"
+    cargo doc --workspace --features "{{all_features}}"
 
 # Runs cargo-deadlinks on the docs
 check-links: generate-docs
-    cargo deadlinks
+    cargo deadlinks --dir target/doc/embedded_graphics
+    cargo deadlinks --dir target/doc/embedded_graphics_core
 
 #----------------------
 # README.md generation
 # ---------------------
 
 # Generate README.md for a single crate
-generate-readme: (_build-readme)
+generate-core-readme: (_build-readmes)
```

Comment:
`_build-readme` had an argument before the other crates were split from the e-g repo: https://github.com/embedded-graphics/embedded-graphics/blob/906ee1180cf31872b03aee037b9022c5fde30630/justfile#L111